ldelf/ta_elf_rel.c

```diff
@@ -478,7 +478,7 @@ static void e32_relocate(struct ta_elf *elf, unsigned int rel_sidx)
 	}
 }
 
-#ifdef ARM64
+#if defined(ARM64) || defined(RV64)
```

Comment:
can you update also line 726 and 732?   s/`ARM64`/`ARM64 || RV64`/

Reply:
ok